4917_Sprint/9-Haoran/Demo/.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add gitignore and don't commit .idea
.idea contains machine/IDE specific settings so if I would pull your project it could break my whole setup